python/ray/dag/compiled_dag_node.py

```diff
@@ -464,6 +480,14 @@ def __init__(
         # this DAG, if any.
         self._nccl_group_id: Optional[str] = None
 
+        # Creates the driver actor. See the comments in the CompiledDAG.DriverActor
+        # class above for more details.
```

Comment:
Could you instead put a shorter comment here? ("See above" is not very robust to later code changes)